lib/order.rb

```diff
@@ -0,0 +1,39 @@
+# require 'send_sms'
+class Order
+
+	def initialize
+		# @ordered_dish_list = []
+		@order_line_list = []
+	end
+
+	def add_dish(dish,qty)
+		order_line = Order_line.new(dish,qty)
+		@order_line_list << order_line
+		# @ordered_dish_list << dish
+	end
+
```

Comment:
Bad idea to create a new OrderLine from within the order as it couples your classes.